README.md

```diff
@@ -3,6 +3,18 @@
 This repository contains templates used to create refernce documentation for developers.google.com,
 cloud.google.com, and other sites.
 
+## Developing
+
+### Testing
+
+You can run the tests with the following command:
+
+```
+TRAMPOLINE_BUILD_FILE=./ci/run_tests.sh TRAMPOLINE_IMAGE=gcr.io/cloud-devrel-kokoro-resources/docfx ci/trampoline_v2.sh
+```
+
+The Dockerfile for the `docfx` image
+[is in googleapis/doc-pipline](https://github.com/googleapis/doc-pipeline/blob/master/docfx/Dockerfile).
```

Comment:
How does this get updated?

Reply:
From `doc-pipeline`. Trampoline V2 will update the image whenever you update the Docker image.

Reply:
To be clear, trampoline script only uploads the image after successful periodic builds:
https://github.com/googleapis/doc-pipeline/blob/master/ci/periodic.cfg

If you prefer always having changes immediately take effect, you can do it by having the Dockerfile in this repo, but I think this repo will be fine with the current setup.